code/__DEFINES/machines.dm

```diff
@@ -12,6 +12,10 @@
 #define IDLE_POWER_USE 1
 #define ACTIVE_POWER_USE 2
 
+// bitflags for a machine's preferences on when it should start processing
+#define START_PROCESSING_ON_INIT	(1<<0)	// if the machine should start processing on Initialize()
+#define START_PROCESSING_MANUALLY	(1<<1)	// if the machine will start processing in the future, after Initialize(), such as when a certain condition is met
+#define NEVER_PROCESS				(1<<2)	// if the machine has no reason to process

```

Comment:
What's the difference between manual and never, there doesn't seem to be a reason to have both

Reply:
You're right there's no difference, I created it simply to make it clear that the machine is not and should never process. I can drop the never and just use manual instead if you want.

Reply:
Yeah it's not really useful to have both